### ISSUE TITLE
[Vertical Taskbar for Windows 11] Windows notifications being misplaced

### ISSUE THREAD

**Issue by bergamin** [ramensoftware/windhawk-mods#1006]
Mod author: @m417z

## Expected Behaviour:

Notifications should show up near the notification button (bottom)

## Actual Behaviour

Notifications are showing up near the Start menu button (top)

### Mod Behaviour:
![image](https://github.com/user-attachments/assets/7265d7be-a2eb-4a62-a71b-393b8ebc1bbd)

### Native behaviour:
![image](https://github.com/user-attachments/assets/e1811617-2ba5-45ba-ba77-62cc78afbee9)

## Other mods installed in this system

- Taskbar Clock Customization, also by @m417z

**Comment by m417z** [comment]
Good catch, I'll fix it in the next version. Meanwhile, if you want to get this right now, you can fork the mod and replace:
```cpp
    *y = rc.top;
````

With:
```cpp
    if (g_target == Target::StartMenu || g_target == Target::SearchHost) {
        *y = rc.top;
    }
```

Let me know if you need more details about it.

**Comment by bergamin** [comment]
Not as critical of an issue to make me go into the trouble of forking it and trying to understand how it works, but thanks for the suggestion

**Comment by m417z** [comment]
Fixed in version 1.2.4.

**Comment by Venthe** [comment]
@m417z Can confirm that it works

@bergamin Just FYI, you can fork in in app; and edit it there. The line in question is 2422.

It is as easy as `details (main window)->fork (mod details of original)->(change line 2422)->compile mod->exit editing mode`; plus disabling the original
